package.json

```diff
@@ -10,10 +10,15 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "broccoli-caching-writer": "0.5.3",
+    "broccoli": "^0.13.3",
+    "broccoli-caching-writer": "^0.5.3",
     "broccoli-kitchen-sink-helpers": "^0.2.5",
+    "chai": "^1.10.0",
     "es6-module-transpiler": "^0.3.6",
+    "esperanto": "^0.6.3",
     "mkdirp": "^0.5.0",
+    "mocha": "^2.1.0",
+    "rsvp": "^3.0.16",
     "walk-sync": "^0.1.3"
```

Comment:
I'm not referring to the order. They are already listed as `devDependencies`, and they got added to `dependencies`.